articles/Why-julia-is-slowly-replacing-python-for-machine-learning-and-data-science/index.md

```diff
@@ -0,0 +1,66 @@
+<!--more-->
+### What is Julia Programming Language?
+Particularly, there was a lack of a programming language that could perform these tasks at a lightning-fast speed. Thus, they resolved to create a high-performance programming language for machine learning and data science. And that is how [Julia](https://julialang.org/) was introduced. Since its launch in 2012, Julia has become a trendy language among data scientists and mathematicians.
+
+Among the current corporate users of Julia are Aviva and BlackRock. They use this tool to calculate risks and execute time-series analysis, respectively. The Celeste project, which is based on the Julia language set a new scientific record due to its fast speed in cataloguing telescopic data for astronomical objects. The Climate Modeling Alliance, CISCO, QuantEcon, NVIDIA, among others also use Julia. In fact, Julia has become a primary tool in the fields of data science, visualization, machine learning and artificial intelligence.
+
+Julia is a worthy competitor for [Python](https://www.python.org/), particularly in the field of arithmetic coding. A preferred tool for developers for many years, Python is facing a threat of a modern programming language. Julia was developed to solve the challenges that Python has. Companies such as Facebook, Instagram, Spotify, Netflix, ILM, Dropbox, Yahoo!, Google and others use Python in one way or another.
+
+### Julia Language Merits
+
+Julia has many features and resources advantageous to machine-learning and data science. This language was designed with a focus on numerical and scientific computation. Julia&#39;s math-friendly syntax makes it ideal for users of [Matlab](https://www.mathworks.com/products/matlab.html), [Octave](https://www.gnu.org/software/octave/about), [Mathematica](https://www.mathematica.org/), [R](https://www.r-project.org/), among other computing languages and environments. With its own native machine learning libraries, Julia is expected to attract more data scientists in the future. An example of such a library is [Flux](https://justgetflux.com/) and is composed of several model patterns ideal for standard use cases. Flux is entirely written in Julia, meaning that users can implement modifications.
+
+### Python Language Advantages
+
+Unlike Julia, Python is a general-purpose language. Despite not being built purposely for data science, Python offers many advantages to machine-learning and data scientists. Python has existed for around 30 years in which it has established strong relationships with many third-party packages. This has attracted many users. One of the drawbacks associated with Python is speed. Better is that Python is implementing some great improvements, especially to the Python interpreter. Advances in parallel and multi-core processing are intended to make Python easier to speed up.
```

Comment:
This sentence was a bit confusing - was not sure what it was trying to convey:
"Better is that Python is implementing some great improvements, especially to the Python interpreter."